### PR TITLE
doc: Fix/Remove some outdated links to API Docs

### PR DESCRIPTION
**Pull request by marikaner**
Fixed some links pointing to the old core packages + removed some unnecessary links.

### CODE REVIEW

File: docs/js/features/odata/common/function-imports/request-builder.mdx [hunk 1]
```diff
@@ -1,8 +1,9 @@
-The [function imports request builder](pathname:///api/1.28.1/classes/sap_cloud_sdk_core.functionimportrequestbuilderv2) helps build a request for a service operation containing parameters in a type-safe way.
-This time, as an example, the [`Warehouse Outbound Delivery Order Service`](https://api.sap.com/api/API_WHSE_OUTB_DLV_ORDER/resource) is used, because not all the services contain function imports like the `Business Partner Service`.
-The type-safe client for the `Warehouse Outbound Delivery Order Service` can be found in the package `@sap/cloud-sdk-vdm-warehouse-outbound-delivery-order-service`.
+The function import request builder helps build a request for a service operations including their parameters.
+Not all services expose function imports.
+We will use the [`Warehouse Outbound Delivery Order Service`](https://api.sap.com/api/API_WHSE_OUTB_DLV_ORDER/resource) as an example.
```
Comment (marikaner) [inline docs/js/features/odata/common/function-imports/request-builder.mdx:3]:
```suggestion
[We will use the Warehouse Outbound Delivery Order service](https://api.sap.com/api/API_WHSE_OUTB_DLV_ORDER/resource) as an example.
```

Comment (florian-richter) [reply docs/js/features/odata/common/function-imports/request-builder.mdx:3]:
```suggestion
We will use the [Warehouse Outbound Delivery Order service](https://api.sap.com/api/API_WHSE_OUTB_DLV_ORDER/resource) as an example.
```
I don't think "we will use the" should be part of the link. The other changes are good though.

**Review by florian-richter — APPROVED** [review]
LGTM

File: docs/js/features/odata/common/function-imports/request-builder.mdx (hunk 1 above)
Comment (marikaner) [reply docs/js/features/odata/common/function-imports/request-builder.mdx]: True

File: docs/js/features/odata/common/function-imports/etag-function-import.mdx [hunk 2]
```diff
@@ -1,9 +1,10 @@
-The `ETag` handling with the function imports is not integrated.
-Below is an example demonstrating how to make use of the [`withCustomHeaders`](pathname:///api/1.26.1/classes/sap_cloud_sdk_core.functionimportrequestbuilder-1#withcustomheaders) for setting `ETag` by your own.
-In order to use [`versionIdentifier`](pathname:///api/1.28.1/classes/sap_cloud_sdk_core.entityv2#versionidentifier) for `ETag` value, make sure you fetch the entity information via e.g., a `get` request.
+Function and action imports do not support `ETag` handling automatically.
+If your service requires the version identifier of an entity to be present, you can pass it to the request on your own using the `addCustomHeaders` method (see example below).
+With this approach, you have to retrieve the required entity with a `getByKey` request, before executing the function import.
```
Comment (marikaner) [inline docs/js/features/odata/common/function-imports/etag-function-import.mdx:3]:
```suggestion
With this approach, you have to retrieve the required entity with a GET request, before executing the function import.
```

**Pull request merged**